VaticanBAV/borget/BAVborget9.xml

```diff
@@ -72,8 +72,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                       <msItem xml:id="ms_i4.2">
                         <locus from="54v" to="55r"/>
-                        <title>First Miracle</title>
-                        <!-- Cf. LIT3053RepCh333 -->
+                        <title>First Miracle</title><!-- Cf. LIT3053RepCh333 -->
```

Comment:
it might be helpful to introduce `<div>`s with derived IDs and labels + incipits in the work record (which I assume is LIT4315Miracles), with incipits, and then here the `@ref` can be added to the `<title>`